crate/operator/main.py

```diff
@@ -565,3 +568,31 @@ async def cluster_update(
                     webhook_scale_payload,
                     logger,
                 )
+
+
+@kopf.on.resume(API_GROUP, "v1", RESOURCE_CRATEDB)
+async def update_cratedb_resource(
+    namespace: str, name: str, spec: kopf.Spec, **kwargs,
+):
+    if "users" in spec:
+        for user_spec in spec["users"]:
+            async with ApiClient() as api_client:
```

Comment:
I'd move the context manager out of the loop, as that prevents the repeated instantiation of the client.